### DISCUSSION TITLE
Added chart property to display chart description by default

### DISCUSSION THREAD

**Discussion by jvani** [apache/superset#19179] [Ideas]
**Is your feature request related to a problem? Please describe.**
Our audience is not necessarily familiar with Superset (i.e., may not know to toggle chart descriptions on/off), but we must convey supplementary information for our audience to best understand our charts.

**Describe the solution you'd like**
An added checkbox/boolean to the "Edit Chart Properties" to either display the chart description by default or not.

**Describe alternatives you've considered**
* Adding text as part of our dashboards. However, the text is not explicitly associated with chart and cannot be customized with CSS (as part of the chart).

**Additional context**
n/a

**Comment by rubypollev** [comment]
Having to explain to every stakeholder to "click the 3-dot menu -> toggle on the chart description" defeats the self-service nature of a dashboard.
